controller.py

```diff
@@ -13,6 +14,8 @@ def __init__(self, networkManager: Optional[any] = None):
     self.networkManager = networkManager
     self.variables = {}
     self.actions = {}
+    t = threading.Thread.__init__(self)
```

Comment:
To invoke a constructor in Python, all you need to do is call the class's name and omit self. Ie: `t = threading.Thread()`.